sqlalchemy_ibmi/backend.py

```diff
@@ -1,3 +1,4 @@
+""" This module specifies the specific dialect for the AS400 using the PyODBCConnector"""
```

Comment:
```suggestion
""" This module specifies the the IBM i dialect using the PyODBCConnector"""
```

Reply:
We should not be adding any new comments/code/whatever that uses the term AS400.